pkg/hooks/hooks.go

```diff
@@ -57,6 +58,9 @@ func New(ctx context.Context, directories []string, extensionStages []string) (m
 	for _, dir := range directories {
 		err = ReadDir(dir, manager.extensionStages, manager.hooks)
 		if err != nil {
+			if os.IsNotExist(err) {
```

Comment:
Can you just do 
if err != nil && !os.IsNotExists(err) {
}

Reply:
Fixed